web/src/resize.js

```diff
@@ -14,20 +14,21 @@ import * as util from "./util";
 function get_new_heights() {
     const res = {};
     const viewport_height = message_viewport.height();
-    const top_navbar_height = $("#top_navbar").safeOuterHeight(true);
-    const right_sidebar_shortcuts_height = $(".right-sidebar-shortcuts").safeOuterHeight(true) || 0;
+    const top_navbar_height = $("#top_navbar").outerHeight(true) ?? 0;
+    const right_sidebar_shortcuts_height = $(".right-sidebar-shortcuts").outerHeight(true) ?? 0;
 
     res.bottom_whitespace_height = viewport_height * 0.4;
 
     res.main_div_min_height = viewport_height - top_navbar_height;
 
     res.stream_filters_max_height =
         viewport_height -
-        Number.parseInt($("#left-sidebar").css("marginTop"), 10) -
-        Number.parseInt($(".narrows_panel").css("marginTop"), 10) -
-        Number.parseInt($(".narrows_panel").css("marginBottom"), 10) -
-        $("#global_filters").safeOuterHeight(true) -
-        $("#private_messages_sticky_header").safeOuterHeight(true);
+            Number.parseInt($("#left-sidebar").css("marginTop"), 10) -
+            Number.parseInt($(".narrows_panel").css("marginTop"), 10) -
+            Number.parseInt($(".narrows_panel").css("marginBottom"), 10) -
+            $("#global_filters").outerHeight(true) ??
+        0 - $("#private_messages_sticky_header").outerHeight(true) ??
+        0;
```

Comment:
This is still wrong due to precedence.

Reply:
@andersk Welp, i ctrl+f'd in vscode for `?? 0` changes, but here it moved to new line so didn't match. 

Sorry for the inconvenience. I'll fix it right away.

Reply:
@andersk fixed